gateway/src/api/latest.rs

```diff
@@ -659,30 +672,32 @@ pub mod tests {
             .await
             .unwrap();
 
-        world.set_super_user("trinity");
-
-        router
-            .call(get_project("reloaded").with_header(&authorization))
-            .map_ok(|resp| assert_eq!(resp.status(), StatusCode::OK))
-            .await
-            .unwrap();
-
-        router
-            .call(delete_project("reloaded").with_header(&authorization))
-            .map_ok(|resp| {
-                assert_eq!(resp.status(), StatusCode::OK);
-            })
-            .await
-            .unwrap();
-
-        // delete returns 404 for project that doesn't exist
-        router
-            .call(delete_project("resurrections").with_header(&authorization))
-            .map_ok(|resp| {
-                assert_eq!(resp.status(), StatusCode::NOT_FOUND);
-            })
-            .await
-            .unwrap();
+        // TODO: setting the user to admin here doesn't update the cached token, so the
```

Comment:
We don't actually have an API for upgrading a user to the admin `AccountTier`, so in a way this test is testing functionality we don't have. We should either add this functionality before releasing the `auth` crate and then fix this test, or make a note of this and make sure we invalidate the cached token when we add the endpoint for changing account tiers.

Reply:
Commenting out the test is good for now. Thanks O